src/NServiceBus.Transport.Msmq/Scripts/NServiceBus.Transport.Msmq.targets

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
```

Comment:
All imported targets/props files should have

```
 <PropertyGroup>
    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
  </PropertyGroup>
```

To ensure incremental builds work correctly.

Reply:
Check